### DISCUSSION TITLE
CSS file does not exist at path error

### DISCUSSION THREAD

**Discussion by davidldear** [duncanmcclean/cookie-notice#75] [Help] [answered]
Looking forward to using the Statamic Add-on but getting an error when trying to implement.  I have done the composer installation and published both the config and assets.  This is a fairly standard mac with statamic and valet setup. I have included a partial component into my layout that contains the tag {{ cookie_notice }}, but get an error on the front-end: `File does not exist at path /Users/daviddear/Projects/peoples/peoplescompany/public/vendor/cookie-notice/http://[::1]:5173/resources/css/cookie-notice.css.`  The cookie notice vendor files are in the /public/vendor folder.  Do you have any ideas or advice?
<img width="1219" alt="Screen Shot 2023-08-03 at 1 10 13 PM" src="https://github.com/duncanmcclean/cookie-notice/assets/729592/dfb738b2-e39c-4fc2-addc-683dfb2c25c3">
<img width="840" alt="Screen Shot 2023-08-03 at 1 10 41 PM" src="https://github.com/duncanmcclean/cookie-notice/assets/729592/91297894-b42d-4757-a641-8a70c053779a">
<img width="823" alt="Screen Shot 2023-08-03 at 1 10 49 PM" src="https://github.com/duncanmcclean/cookie-notice/assets/729592/ff93fc67-67ac-4d5e-8794-223391517611">

**Comment by duncanmcclean** [comment, marked as answer]
Hey!

I was unable to replicate this issue locally but I've just [tagged a release](https://github.com/duncanmcclean/cookie-notice/releases/tag/v8.1.1) which *might* fix the issue. Would you be able to give it a go?

**Reply by davidldear** [reply]
That did indeed solve the issue. Thank you very much!

**Reply by duncanmcclean** [reply]
Awesome, glad to hear it!
